base/applications/cmdutils/help/lang/ro-RO.rc

```diff
@@ -1,8 +1,9 @@
 /*
- * FILE:       base/applications/cmdutils/help/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * AUTHOR:     Magnus Olsen, 2005
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS DX Diagnostic
```

Comment:
This has absolutely nothing to do with "ReactOS DX Diagnostic". A good example, why it is a foolish idea to fill all of our rcs with obsolete information by duplicating wrong data into 36 or more rc files.

Reply:
Sorry, I copied the header on my computer and I forgot to change the name when I pasted into the translated file from PR. I will modify it right now.

Reply:
No problem. It is obvious how copy-paste-issues do evolve. Clever programmers do apply workflows therefore that do minimize the need for copy-n-paste. Again that comment is not so much targetting you (just following orders you received), but to the "ordererers".